index.js

```diff
@@ -23,6 +23,29 @@ require("core-js/fn/number/to-fixed");
 require("core-js/fn/number/to-precision");
 require("core-js/fn/parse-float");
 require("core-js/fn/parse-int");
+require("core-js/fn/array/from");
+require("core-js/fn/array/of");
+require("core-js/fn/array/is-array");
+require("core-js/fn/array/iterator");
+require("core-js/fn/array/copy-within");
+require("core-js/fn/array/fill");
+require("core-js/fn/array/find");
+require("core-js/fn/array/find-index");
+require("core-js/fn/array/values");
+require("core-js/fn/array/keys");
+require("core-js/fn/array/entries");
+require("core-js/fn/array/slice");
+require("core-js/fn/array/join");
+require("core-js/fn/array/index-of");
+require("core-js/fn/array/last-index-of");
+require("core-js/fn/array/every");
+require("core-js/fn/array/some");
+require("core-js/fn/array/for-each");
+require("core-js/fn/array/map");
+require("core-js/fn/array/filter");
+require("core-js/fn/array/reduce");
+require("core-js/fn/array/reduce-right");
+require("core-js/fn/array/sort");
```

Comment:
If you're not omitting any polyfills, I think you can just `require("core-js/fn/array")` to import everything.